docs/build/program/apis/cloud.md

```diff
@@ -1332,3 +1332,262 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 {{% /tab %}}
 {{< /tabs >}}
+
+### CreateOrganizationInvite
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Create an organization invite and send it by email.
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address to send the invite to.
+- `authorizations` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Specifications of the
+  authorizations to include in the invite.
```

Comment:
```suggestion
- `authorizations` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Specifications of the authorizations to include in the invite.
```
One line I think?

Also, can you add an example or two of what a "specification" would be? Or is that not practical and the user should just click the link for more info (fine if that's the case)?

Reply:
It just means those objects. I'll remove the specifications phrasing